ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/ViewVulnReport/JobDetails.tsx

```diff
@@ -10,20 +10,20 @@ import {
 } from '@patternfly/react-core';
 
 import { getDateTime } from 'utils/dateUtils';
-import { ReportSnapshot } from 'services/ReportsService.types';
+import { ReportStatus } from 'services/ReportsService.types';
 import { getReportStatusText } from '../utils';
 
 export type JobDetailsProps = {
-    reportSnapshot: ReportSnapshot;
+    reportStatus: ReportStatus;
+    isDownloadAvailable: boolean;
 };
 
-function JobDetails({ reportSnapshot }: JobDetailsProps): ReactElement {
-    const { isDownloadAvailable, reportStatus } = reportSnapshot;
+function JobDetails({ reportStatus, isDownloadAvailable }: JobDetailsProps): ReactElement {
     const { reportRequestType, completedAt } = reportStatus;
     return (
         <Flex direction={{ default: 'column' }}>
             <FlexItem>
-                <Title headingLevel="h3">Job details</Title>
+                <Title headingLevel="h2">Job details</Title>
```

Comment:
Does `h3` versus `h2` depend on context?

For compliance scan schedules, wizard review step needs `h3` but detail page needs `h2` level.

Reply:
Fixed. Details sections will use `h2` and the wizard review step will use `h3`